app/src/main/java/com/brewfind/ui/main/MainFragment.kt

```diff
@@ -29,4 +25,8 @@ class MainFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
```

Comment:
Moved companion object to proper location (bottom)
Source: https://kotlinlang.org/docs/coding-conventions.html#source-file-names 
found under 'class layout' section